code/modules/mob/living/living.dm

```diff
@@ -1106,6 +1106,7 @@
 
 /mob/living/onTransitZ(old_z,new_z)
 	..()
+	SEND_SIGNAL(src, COMSIG_ASHWALKER_LEGS)
```

Comment:
Can you please put this down in the parent call, and instead of a new comsig, can you use `COMSIG_MOVABLE_Z_CHANGED`

Reply:
Is there a reason to do it there? You cant get ashwalker speed without being a `/living` type, so why check that every single transit?

Reply:
We want this signal to be applicable to everything, and hence the name change too.

Signal names are like proc names rather than trait names.

its like naming a proc `ashwalker_legs()` and having use it for some time anyone wants to have something happen when someone transfers z-levels, including non-ashwalker leg code.

Reply:
Ah, I see. Future proofing. Forgot signals arent proc-connected since the proc is only called on the registered signal. Was looking at it more like a trait.

Will swap.